verl/workers/reward_manager/prime.py

```diff
@@ -48,7 +48,7 @@ async def single_compute_score(evaluation_func, completion, reference, task, tas
 
 async def parallel_compute_score_async(evaluation_func, completions, references, tasks, extra_info=None, num_processes=64):
     scores = []
-    with ProcessPoolExecutor(max_workers=num_processes) as executor:
+    with ThreadPoolExecutor(max_workers=num_processes) as executor:
```

Comment:
ThreadPoolExecutor is not a good idea since compute_score is compute intensive, python GIL prevents it to use multi cores.